mlf_core/create/templates/mlflow/mlflow_pytorch/{{ cookiecutter.project_slug }}/environment.yml

```diff
@@ -5,14 +5,17 @@ channels:
   - pytorch
 dependencies:
   - defaults::cudatoolkit=10.1
-  - defaults::python=3.7
-  - defaults::click=7.1.2
-  - conda-forge::rich=9.3.0
-  - conda-forge::mlflow=1.12.1
+  - defaults::python=3.8.2
   - conda-forge::tensorboardx=2.1
   - pytorch::pytorch=1.6.0=py3.7_cuda10.1.243_cudnn7.6.3_0
   - pytorch::torchvision=0.7.0=py37_cu101
+  - pytorch-lightning==1.0.2
+  - torchvision=0.7.0
   - pip
   - pip:
     - pycuda==2019.1.2  # not on Conda
-    - system-intelligence==2.0.1  # not yet on Conda
+    - mlflow==1.13.1
+    - cloudpickle==1.6.0
+    - boto3==1.16.58
+    - system-intelligence==2.0.1
+    - rich==9.6.1
```

Comment:
rich is already at 9.8.1 or something

Reply:
Its incompatible with system-intelligence, as the version there is pinned to 9.6.1

Reply:
Ahhh.
I suggest that you release a system-intelligence version aka 2.0.2 with rich pinned to

>=9.8.2